### PR TITLE
NR-38000-Blazor Web Assembly quickstart.

### PR DESCRIPTION
**Pull request by pkudikyala**
# Summary

**Ticket Url:** https://issues.newrelic.com/browse/NR-38000
**Description:** This PR has the new quickstart integration named "Blazor Web Assembly"

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

# Screenshots:

screencapture-newrelic-instant-observability-preview-2022-12-09-16_01_11

### CODE REVIEW

File: quickstarts/blazor-web-assembly/config.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    description: |
+      Our Browser Monitoring agent can be used to instrument and monitor your Blazor Web Assembly application. Perform an immediate analysis of your performance statistics. 
+    url: >- 
+      https://docs.newrelic.com/
+keywords:  
+  - apm
+  - dotnet
+  - .net
+  - blazor
+  - c-sharp
```
Comment (pkosanam) [inline quickstarts/blazor-web-assembly/config.yml:47]: can remove

File: quickstarts/blazor-web-assembly/config.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - c#
```
Comment (pkosanam) [inline quickstarts/blazor-web-assembly/config.yml:48]: can remove

File: quickstarts/blazor-web-assembly/config.yml (hunk 1 above)
Comment (pkudikyala) [reply quickstarts/blazor-web-assembly/config.yml]: Hi @pkosanam, removed. Kindly review it.

File: quickstarts/blazor-web-assembly/config.yml (hunk 2 above)
Comment (pkudikyala) [reply quickstarts/blazor-web-assembly/config.yml]: Hi @pkosanam, removed. Kindly review it.

File: dashboards/blazor-web-assembly/blazor-web-assembly.json [hunk 3]
```diff
@@ -0,0 +1,1158 @@
...
+						"row": 1,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "![NuxtJS icon](https://www.zenesys.com/getattachment/Blog/Web-Development/January-2022/Blazor-Server-vs-Blazor-Web-Assembly/MicrosoftTeams-image-(15).png?width=400&height=300)\n"
```
Comment (pkosanam) [inline dashboards/blazor-web-assembly/blazor-web-assembly.json:23]: please update the text here from "NuxtJS icon" to "Blazor Web Assembly icon"

File: dashboards/blazor-web-assembly/blazor-web-assembly.json [hunk 4]
```diff
@@ -0,0 +1,1158 @@
...
+						"row": 4,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/).\n\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=BlazorWebAssembly) here and let us know how we can improve it for you."
```
Comment (pkosanam) [inline dashboards/blazor-web-assembly/blazor-web-assembly.json:120]: please update the "Add Data" link to "https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/browser-monitoring-integrations/blazor-webassembly/"

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: dashboards/blazor-web-assembly/blazor-web-assembly.json (hunk 3 above)
Comment (pkudikyala) [reply dashboards/blazor-web-assembly/blazor-web-assembly.json]: Hi @pkosanam,
Changed icon name. Kindly review it

File: dashboards/blazor-web-assembly/blazor-web-assembly.json (hunk 4 above)
Comment (pkudikyala) [reply dashboards/blazor-web-assembly/blazor-web-assembly.json]: Hi @pkosanam,
Updated doc's URL in the dashboard. Kindly review it

**Review by pkosanam — APPROVED** [review]
looks good

**Comment by MichelLosier** [comment]
Updated preview link: https://newrelic.com/instant-observability/preview?pr=1520&quickstart=blazor-web-assembly

**Review by MichelLosier — APPROVED** [review]
Everything looks good here, :shipit:

File: alert-policies/blazor-web-assembly/ApplicationErrorRate.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Name of the alert
+name: Application Error Rate
+
+# Description and details
+description: |+
+  This alert is triggered when the application error rate are higher than 5 for 5 minutes.
```
Comment (paperclypse) [inline alert-policies/blazor-web-assembly/ApplicationErrorRate.yml:6]: "This alert is triggered when the application error rates are higher than 5 for 5 minutes."

File: install/blazor-web-assembly/install.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+id: blazor-web-assembly
+
+name: Blazor Web Assembly
```
Comment (paperclypse) [inline install/blazor-web-assembly/install.yml:3]: According to the Blazor site, it should be "Blazor WebAssembly"

File: install/blazor-web-assembly/install.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+title: Blazor Web Assembly
```
Comment (paperclypse) [inline install/blazor-web-assembly/install.yml:5]: "Blazor WebAssembly"

File: install/blazor-web-assembly/install.yml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+description: Instrument and monitor your Blazor-built app with our .NET agent. Analyze your performance data right out of the box.
```
Comment (paperclypse) [inline install/blazor-web-assembly/install.yml:6]: "Instrument and monitor your Blazor-built app with our .NET APM agent."

File: quickstarts/blazor-web-assembly/config.yml [hunk 9]
```diff
@@ -0,0 +1,53 @@
+
+
+slug: blazor-web-assembly
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Web Assembly
```
Comment (paperclypse) [inline quickstarts/blazor-web-assembly/config.yml:5]: Blazor WebAssembly

File: quickstarts/blazor-web-assembly/config.yml [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+  Microsoft developed Blazor Web Assembly to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Web Assembly integration with the Browser Monitoring agent lets you monitor your apps with our quickstart dashboards right out of the box.
```
Comment (paperclypse) [inline quickstarts/blazor-web-assembly/config.yml:7]: Blazor WebAssembly

Comment (paperclypse) [reply quickstarts/blazor-web-assembly/config.yml:7]: There's this, to me, frustrating thing that these quickstart pages do. I don't think we need to tell someone what Blazor WebAssembly is to people on the New Relic I/O page. They're here to learn about what New Relic can do for them!

How about something more like: 

"As you probably know, Microsoft developed Blazor WebAssembly to build web UI via C#, HTML, and CSS. With our browser monitoring agent, you can monitor your Blazor WebAssembly app performance with our pre-built dashboards." or something.

File: quickstarts/blazor-web-assembly/config.yml [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+
+  ## Why monitor Blazor Web Assembly?
```
Comment (paperclypse) [inline quickstarts/blazor-web-assembly/config.yml:10]: The text in this section is all about why I should Blazor WebAssembly *not* New Relic. This whole section needs to be rewritten to answer the question "Why should I monitor my Blazor WebAssembly app with New Relic?"

File: quickstarts/blazor-web-assembly/config.yml [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+  Blazor Web Assembly offers a lot of useful features in addition to allowing C# developers to utilize their preferred language for web development. Blazor Web Assembly allows you to run your code through WebAssembly on the server or the client, and you can decide which users or devices experience each method.
+  
+  There are a few compelling arguments that support the employment of Blazor Web Assembly:
+  - All of the major browsers support WebAssembly.
+  - Using C# for interactive web apps.
+  - Utilize current C# libraries.
+  - Near-native performance.
+  - Debugging and tooling.
+
+  ## What’s included in this quickstart?
```
Comment (paperclypse) [inline quickstarts/blazor-web-assembly/config.yml:21]: Shorten to "What's included?"

File: quickstarts/blazor-web-assembly/config.yml [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+
+  Get these Blazor Web Assembly monitoring features out of the box with our quickstart:
```
Comment (paperclypse) [inline quickstarts/blazor-web-assembly/config.yml:23]: Delete "with our quickstart"

File: quickstarts/blazor-web-assembly/config.yml [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+  - Understand the health of your system with dashboards that show throughput and error rate data, logs, web transaction time, page views, page load time, and more. 
+  - Alerts.
```
Comment (paperclypse) [inline quickstarts/blazor-web-assembly/config.yml:25]: I think we need to expand on "Alerts" :D

"Get notified when your app suffers critical failures"

**Pull request merged**